status/projects.py

```diff
@@ -173,6 +190,9 @@ def project_summary_data(self, row):
             queued = row.value["queued"]
             diff = now - dateutil.parser.parse(queued)
             row.value["days_in_production"] = diff.days
+            field_sources["days_in_production"] = (
```

Comment:
Same here, if it could be defined in a single place

Reply:
Updated this as well.